### ISSUE TITLE
Bad interaction on timing with letsencrypt.org

### ISSUE THREAD

**Issue by leighklotz** [RaymiiOrg/certificate-expiry-monitor#10]
letsencrypt certificate rotation is 90 days, so if you use letsencrypt, you'll always get certification expiration warnings that aren't meaningful because of the auto-renewal jobs.  Being able to set this to something shorter would be good.

**Comment by thefinn93** [comment]
Yeah, I have this issue too. Ideally, there could be some sort of option to notify if the cert is going to expire in 4 days or less (because the default Let's Encrypt client installs the new cert 5 days before expiry of the old one)

**Comment by bfred-it** [comment]
LE reportedly renews certificates if they expire ["in less than 30 days."](https://community.letsencrypt.org/t/how-often-should-i-run-the-cerbot-cron-job-to-update-the-certificates/18851/2)

Given that [certbot.eff.org recommends](https://certbot.eff.org/#ubuntutrusty-apache) running the auto-renewal attempt twice per day, if a LE certificates expires in 29 or 28 days the renewal likely failed.

Therefore "28 days or less" would be a good time to start sending emails

**Comment by ptman** [comment]
Maybe skip the 90 day notice if the cert is issued by letsencrypt?
